.travis.yml

```diff
@@ -104,6 +110,9 @@ matrix:
       env: DESCRIPTION="beta Rust release"
 
     - rust: nightly
+      os:
+        - linux
+        - osx
```

Comment:
This could cause a problem: it does a doc-upload at the end. Perhaps you can prefix that:
```
test "$TRAVIS_OS_NAME" = "linux" && travis-cargo --only nightly doc-upload
```